tests/unit/test_logging/test_logging_config.py

```diff
@@ -82,6 +83,12 @@ def test_picologging_dictconfig_when_disabled(dict_config_mock: Mock) -> None:
         assert not dict_config_mock.called
 
 
+def test_get_logger_without_logging_config() -> None:
+    with create_test_client(logging_config=None) as client:
+        with pytest.raises(ImproperlyConfiguredException):
```

Comment:
Could you add a `match` to make sure the exception message is the logging one? Just to confirm that the error is from the placeholder function.